pkg/tcl/apitcl/v1/testworkflows.go

```diff
@@ -305,11 +318,27 @@ func (s *apiTCL) ExecuteTestWorkflowHandler() fiber.Handler {
 			return s.BadRequest(c, errPrefix, "processing error", err)
 		}
 
-		now := time.Now()
+		// Load execution identifier data
+		// TODO: Consider if that should not be shared (as now it is between Tests and Test Suites)
+		number, _ := s.ExecutionResults.GetNextExecutionNumber(context.Background(), workflow.Name)
+		executionName := request.Name
```

Comment:
will be a bit confusing, if to use it together with tests

Reply:
It is, but at the moment the `GetNextExecutionNumber` for tests and test suites is shared to - `sequences` do not have differentation between types.